helm/hpcc/values.yaml

```diff
@@ -697,10 +697,11 @@ esp:
     #labels:
     #  mylabel: "3"
     ## Links specify the web links for a service. The web links may be shown on ECLWatch.
-    #links:
-    #- name: linkname
-    #  description: "some description"
-    #  url: "http://abc.com/def?g=1"
+    links:
```

Comment:
@GordonSmith this block announces external links for EclWatch to embed. I've added a sample url mapping to a specific trace id which eclwatch would need to inject. I added the "linkedResource" attribute for you to know what type of resource to attach this to.